.github/action-inputs-config.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "name": "speakeasy_version",
+    "validationRegex": "/^[\\w.\\-]+$/.source",
+    "validationMessage": "Letters, numbers, or .-_ only"
+  },
+  {
+    "name": "openapi_doc_location",
+    "validationRegex": "/(^(https?):\\/\\/([\\w\\-]+\\.)+\\w+(\\/.*)?$)?^[\\w.\\-\\/]+$/i.source",
+    "validationMessage": "Must be a valid server URL or file path containing letters, numbers, or .-_/ only"
+  },
+  {
+    "name": "openapi_doc_auth_header",
+    "validationRegex": "/^[A-Za-z\\-]+$/.source",
+    "validationMessage": "Letters or - only"
```

Comment:
I need to see what the possible chars are for all these auth fields and make the `validationRegex` and `validationMessage` more specific accordingly.